docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
+Options:
+
+- `-o | --output` - The output file. If omitted, overwrites input. Must be a directory if multiple files are specified.
+- `-b | --base-directory` - Base directory for files to override the working directory.
+- `-f | --force` - Overwrites a signature if it exists.
+- `-m | --max-concurrency` - Maximum concurrency (default is 4)
+- `-fl | --filelist` - Path to file containing paths of files to sign within an archive
+- `-fd | --file-digest` - The digest algorithm to hash the file with.
+- `-tr | --timestamp-rfc3161` - Specifies the RFC 3161 timestamp server's URL.
+- `-d | --description` - Description
+- `-u | --descriptionUrl` - Description Url
+- `-td | --timestamp-digest` - Used with the -tr switch to request a digest algorithm used by the RFC 3161 timestamp server.
+- `-kvu | --azure-key-vault-url` - The URL to an Azure Key Vault.
+- `-kvt | --azure-key-vault-tenant-id` - The Tenant Id to authenticate to the Azure Key Vault..
+- `-kvi | --azure-key-vault-client-id` - The Client ID to authenticate to the Azure Key Vault.
+- `-kvs | --azure-key-vault-client-secret` - The Client Secret to authenticate to the Azure Key Vault.
+- `-kvc | --azure-key-vault-certificate` - The name of the certificate in Azure Key Vault.
+- `-kvm | --azure-key-vault-managed-identity` - Use a Managed Identity to access Azure Key Vault.
```

Comment:
Are we missing `-h | --help` option here?

Reply:
Could be missing in the docs; I believe it's there by default with system.commandline